serverless.yml

```diff
@@ -123,7 +123,19 @@ stepFunctions:
               - ErrorEquals:
                   - States.ALL
                 Next: errorHandler
-            Next: defaultSuccess
+            Next: loadDiscreteGW
+          loadDiscreteGW:
+            Type: Task
+            Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-capture-discrete-loader-${self:provider.stage}-loadDiscrete
+            Parameters:
+              fieldVisitIdentifiers.$: $.fieldVisitIdentifiers
```

Comment:
I believe this becomes the only input that makes it to the lambda... I was under the understanding that a lambda function should get the full input and grab what it needs within the code. :thinking:

Reply:
I was wondering about how to do this... it makes me wonder: do we actually need the parameters defined in the `fieldVisitTransform` state a few lines above this? https://github.com/usgs/aqts-capture-state-machine/blob/master/serverless.yml#L117 ?